### DISCUSSION TITLE
css-loader seems not to load icss?

### DISCUSSION THREAD

**Discussion by LunaticMuch** [webpack/webpack#16914] [Q&A]
If not mistaken, using css-loader with icss I should be able to define a css file as

```
:export {
  iconSize: '24px';
}
```

then in my `otherfile.tsx` I could have

```
import * as _ from './variables.css';
console.log(_.iconSize)
```

But this actually does not work, as the `iconSize` does not exist.
In my webpack config I have

```
{
  loader: 'css-loader',
  options: {
    modules: {
      auto: true,
      exportGlobals: true,
      namedExport: true,
      mode: 'icss',
    },
    sourceMap: true,
  },
```

Which apparently does not go anywhere. I get 
`export 'iconSize' (imported as '_') was not found in './variables.css' (module has no exports)`

Am I thinking wrong about icss and css-loader?

**Comment by alexander-akait** [comment]
> export 'iconSize' (imported as '_') was not found in './variables.css' (module has no exports)

How you use it? Can you show me an import?

**Comment by LunaticMuch** [comment]
@alexander-akait I tried different way:

```
import * as _ from './variables.css';
console.log(_.iconSize)
```

or in this way

```
import iconSize from './variables.css';
console.log(iconSize)
```

and still I get 

`export 'default' (imported as 'iconSize') was not found in './variables.css' (module has no exports)`


If I do this
```
import * as _ from './variables.css';
console.log(_)
```

I have no error, but the console gets the dump of an entire EsModule.
I am on `"css-loader": "6.7.1"`

**Reply by LunaticMuch** [reply]
Fixed.
So I had to load separately the variables from other css. 
It now works!

**Reply by alexander-akait** [reply]
Great, if you will have any questions, feel free to ask me
